pfio/NetCDF4_get_var.H

```
@@ -36,10 +36,20 @@
 
      _ASSERT(status==0,"Variable not found: "//trim(var_name)//" in file: "//trim(this%origin_file))
      !$omp critical
+#if (_VARTYPE == 0)
+
+#if (_RANK == 1)
+    ! only support rank 1 of string
+    status = pfio_nf90_get_var_string(ncid, varid, values, start, count)
```

Should we raise an error if rank is different?